tetris.py

```diff
@@ -86,6 +86,7 @@ def run_gui():
     root.bind("<Left>", tetris_gui.move_block_left)
     root.bind("<Right>", tetris_gui.move_block_right)
     root.bind("<Up>", tetris_gui.rotate_block)
+    root.bind('<P>', tetris_gui.pause_game)
```

Comment:
you probably want to trigger when p is pressed, not shift+p:

```suggestion
    root.bind('<p>', tetris_gui.pause_game)
```